### PR TITLE
Readme contributelinks

### PR DESCRIPTION
**Pull request by fsoppelsa**
Add the Swarm contribute link.
Fix a not referenced "here" link.
    
Signed-off-by: Fabrizio Soppelsa <fsoppelsa@mirantis.com>

### CODE REVIEW

**Comment by abronan** [comment]
Thanks for the contribution @fsoppelsa.

Could you please squash your commits? Thanks!

/cc @thaJeztah @moxiegirl

**Comment by fsoppelsa** [comment]
Yes, sorry, I didn't notice :)

File: README.md [hunk 1]
```diff
@@ -73,13 +73,13 @@ replacing `docker run swarm` with just `swarm`.
 
 ## Participating
 
-You can contribute to Docker Swarm in several different ways:
+You can [contribute](https://github.com/docker/swarm/blob/master/CONTRIBUTING.md) to Docker Swarm in several different ways:
```
Comment (thaJeztah) [inline README.md]: Just a nit; I think this can be replaced by simply `CONTRIBUTING.md` (it's in the same repository)

Comment (thaJeztah) [inline README.md]: Actually, let's keep it like this, we want people to always visit the most up-to-date version (in case they visit from an older tag)

**Comment by thaJeztah** [comment]
Thanks @fsoppelsa, LGTM

**Comment by abronan** [comment]
Thanks @fsoppelsa!

LGTM

**Pull request merged**